include/onnxruntime/core/providers/openvino/openvino_provider_factory.h

```diff
@@ -2,22 +2,10 @@
 // Licensed under the MIT License
 
 #include "onnxruntime_c_api.h"
-
 #ifdef __cplusplus
+#include <vector>
+#include <string>
 struct ProviderInfo_OpenVINO {
   virtual std::vector<std::string> GetAvailableDevices() const = 0;
 };
-
-extern "C" {
-#endif
-
-/**
- * \param device_type openvino device type and precision. Could be any of
- * CPU_FP32, GPU_FP32, GPU_FP16, MYRIAD_FP16, VAD-M_FP16 or VAD-F_FP32.
- */
-ORT_API_STATUS(OrtSessionOptionsAppendExecutionProvider_OpenVINO,
```

Comment:
why are these api's being removed for OpenVINO and TensorRT?

Reply:
This is the deprecated one, and it misses features.

Reply:
better to let the code owners handle the deprecation, and ensure smooth transition.
i'll ask Intel how to handle it.